conduit/logging.go

```diff
@@ -0,0 +1,20 @@
+package conduit
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+// PluginLogFormatter formats the log message with special conduit tags
+type PluginLogFormatter struct {
+	Formatter *log.JSONFormatter
+	Type      string
+	Name      string
+}
+
+// Format allows this to be used as a logrus formatter
+func (f PluginLogFormatter) Format(entry *log.Entry) ([]byte, error) {
+	// Underscores force these to be in the front in order type -> name
+	entry.Data["__type"] = f.Type
+	entry.Data["_name"] = f.Name
```

Comment:
Why use 1 underscore in one key and 2 in the other?

Reply:
It's to force the order when outputting the json to the screen to be type then name.  Like this:

`{"__type":"Conduit","_name":"main","level":"info","msg":"Log level set to: warn","time":"2022-08-09T08:48:44-04:00"}`

If one is looking at the logs manually it is a little easier to look at.

Reply:
Looks like you can provide a sorting function for this kind of thing without messing around with the names: https://pkg.go.dev/github.com/sirupsen/logrus#TextFormatter.SortingFunc

Reply:
I think that is only for `TextFormatter` not `JSONFormatter` though.  Do we care if we lose the JSON data format?  I figured the structured logging would be useful for log aggregating tools.